### ISSUE TITLE
Cropped background image on newer Android versions

### ISSUE THREAD

**Issue by igor-cali** [leinardi/UbuntuCountdownWidget#5]
See PR #4.

**Comment by igor-cali** [comment]
Drafting this one.

**Comment by igor-cali** [comment]
@leinardi if you can provide me with vector graphics for `app/src/main/res/drawable-hdpi/background_dark.9.png` and `app/src/main/res/drawable-hdpi/background_light.9.png`, I can include good looking alternatives for new versions.

**Comment by leinardi** [comment]
@igor-cali unfortunately I never got the original vector graphics for those background but, since meanwhile I got some experience with Figma, I quickly recreated them: 84790ff94f8f56a895edc386fa962b1326ff5bbb

I'm not sure if is possible with App Widget but, if it is, I would like to use Vector Drawable instead of 9 Patches since now we have the SVG source. And don't worry if the 2x2 corner now will be bigger, that was actually the desired look but, since I was missing the vector source, I had to fall back to the 1x1 corner size also for that.

Let me know if I can help with something else.
